src/pages/index.js

```diff
@@ -322,3 +328,15 @@ deleteForm.addEventListener("submit", handleDeleteSubmit);
 editFormElement.addEventListener("submit", handleEditFormSubmit);
 cardForm.addEventListener("submit", handleAddCardSubmit);
 avatarForm.addEventListener("submit", handleAvatarSubmit);
+document.addEventListener("keydown", (evt) => {
+  const openModalElement = document.querySelector(".modal_opened");
+  if (openModalElement && evt.key === "Escape") {
+    closeModal(openModalElement);
```

Comment:
Please, check your project 6. You had this `Escape` handling there https://snipboard.io/b40skP.jpg  . You shouldn't have changed it because you need to add and remove the `Escape` listener correctly, according to the task and checklist